src/Security/Group.php

```diff
@@ -618,9 +618,12 @@ public function AllChildrenIncludingDeleted()
     {
         /** @var Hierarchy $extInstance */
         $extInstance = $this->getExtensionInstance(Hierarchy::class);
-        $extInstance->setOwner($this);
-        $children = $extInstance->AllChildrenIncludingDeleted();
-        $extInstance->clearOwner();
+        try {
+            $extInstance->setOwner($this);
+            $children = $extInstance->AllChildrenIncludingDeleted();
+        } finally {
+            $extInstance->clearOwner();
+        }
```

Comment:
Possibly a stupid question, and I know you didn’t write this, but could this be replaced with `$children = parent::AllChildrenIncludingDeleted()`? Won’t that invoke `CustomMethods::__call()`?

Reply:
Yes that shousd I think.